connectors/aliyun/rocketmq-connect-fc/src/main/java/com/aliyun/rocketmq/connect/fc/sink/FcSinkConnector.java

```diff
@@ -0,0 +1,82 @@
+package com.aliyun.rocketmq.connect.fc.sink;
```

Comment:
change package to org.apache.rocketmq.connect.xxx

Reply:
Processed